### ISSUE TITLE
Allow renaming location label for public booking page

### ISSUE THREAD

**Issue by alishaz-polymath** [calcom/cal.com#14061]
### Is your proposal related to a problem?

> Hi, we have two locations configured for several of our event types. These locations include: Phone Call and Zoom Meeting represented by "Attendee Phone Number" and "Organizer's Default App" at the point of configuring the locations.
However, on the actual booking form these locations show as  "Attendee Phone Number" and "Organizer's Default App" as well, and we would like to rename them to "Phone Call" and "Zoom Meeting" respectively.
Please, how may we go about this renaming? We can't seem to find the option to do that.

### Describe the solution you'd like

An option to rename the selected event type location's label which shows up in the public booking page.

##### House rules
- If this issue has a `🚨 needs approval` label, don't start coding yet. Wait until a core member approves feature request by removing this label, then you can start coding.
  - For clarity: Non-core member issues automatically get the `🚨 needs approval` label.
  - Your feature ideas are invaluable to us! However, they undergo review to ensure alignment with the product's direction.

**Comment by CarinaWolli** [comment]
We have this for organizer's default now. Let's create a new issue if we want this for other locations as well
<img width="403" height="181" alt="Image" src="https://github.com/user-attachments/assets/1dbe5596-a624-4257-89b7-c530eae72eda" />
